File: src/main/java/com/haozileung/web/domain/system/Dictionary.java
package com.haozileung.web.domain.system;

import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;

/**
 * Created by haozi on 16-3-7.
 */
public class Dictionary {
    private Long dictionaryId;
    private String code;
    private String name;
    private DictionaryType type;
    private Status status;

    public Long getDictionaryId() {
        return dictionaryId;
    }

    public void setDictionaryId(Long dictionaryId) {
        this.dictionaryId = dictionaryId;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public DictionaryType getType() {
        return type;
    }

    public void setType(DictionaryType type) {
        this.type = type;
    }

    public Status getStatus() {
        return status;
    }

    public void setStatus(Status status) {
        this.status = status;
    }

    @Override
    public String toString() {
        return ToStringBuilder.reflectionToString(this, ToStringStyle.JSON_STYLE);
    }
}

File: src/main/kotlin/com/haozileung/web/domain/dictionary/Dictionary.kt
/*
 * Powered By [rapid-framework]
 * Web Site: http://www.rapid-framework.org.cn
 * Google Code: http://code.google.com/p/rapid-framework/
 * Since 2008 - 2016
 */

package com.haozileung.web.domain.dictionary;

import java.io.Serializable;

/**
 * @author Haozi
 * @version 1.0
 * @since 1.0
 */
data class Dictionary(var dictionaryId: Long? = null,
                      var dicTypeId: Long? = null,
                      var code: String? = null,
                      var name: String? = null,
                      var remarks: String? = null,
                      var status: Int? = null) : Serializable
